src/Perspex.SceneGraph/Media/DashStyle.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace Perspex.Media
+{
+    using Perspex.Animation;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public class DashStyle : Animatable
+    {
+        private static DashStyle dash;
+        public static DashStyle Dash
+        {
```

Comment:
I'm not sure about this style of declaring fields/properties together. I don't have anything against it per-se, but this style isn't used in the rest of the codebase and I'd rather have things consistent.